packages/dds/map/src/directory.ts

```diff
@@ -2248,7 +2248,8 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
 			if (op.type === "createSubDirectory") {
 				const dir = this._subdirectories.get(op.subdirName);
 				// Child sub directory create seq number can't be lower than the parent subdirectory.
-				if (this.sequenceNumber !== -1 && this.sequenceNumber < msg.sequenceNumber) {
+				// The sequence number can be consistent when multiple createSubDirectory occurs with grouped batching enabled.
```

Comment:
Nit. Not sure "consistent" would explain enough to someone without context.
```suggestion
				// The sequence number for multiple ops can be the same when multiple createSubDirectory occurs with grouped batching enabled, thus <= and not just <.
```